resources/rpi/user-data

```diff
@@ -0,0 +1,74 @@
+#cloud-config
+# https://cloudinit.readthedocs.io/
+
+# Set the first part of the name to camera device.
+hostname: huntsmanpi000
+
+# If you have set up ssh on github you can pull down your
+# key automatically so that you can log into the unit without
+# a password. Change 'CHANGEME' to your username below.
+ssh_import_id:
+  - gh:huntsmanscope
+
+chpasswd:
+  expire: true
+  list:
+    - huntsman:huntsman
+
+ssh_pwauth: true
+
+groups:
+  - huntsman
+  - panoptes
+  - docker
+
+users:
+  - name: huntsman
+    gecos: Hunstman User
+    primary_group: huntsman
+    groups: users, admin, dialout, plugdev, docker, i2c, input, gpio, panoptes
+    sudo: ALL=(ALL) NOPASSWD:ALL
+    lock_passwd: false
+    shell: /bin/zsh
+
+## Update apt database and upgrade packages on first boot
+package_update: true
+package_upgrade: true
+
+byobu: enable
+
+ntp:
+  enabled: true
+  servers:
```

Comment:
Might want to put the SSO ntp server in here, top of the list.

Reply:
Actually, I did in the first commit and then got worried @lspitler wouldn't like that. :shushing_face:

Reply:
Oh yeah, IPs in public repos. Not good.